models/trickortreats.js

```diff
@@ -0,0 +1,36 @@
+const low = require('lowdb')
+const FileSync = require('lowdb/adapters/FileSync')
+
+const adapter = new FileSync('db/trickortreats.json')
+const db = low(adapter)
+
+// Set some defaults (required if your JSON file is empty)
+db.defaults({ trickortreats: [] }).write()
+
+// Find a trick or treat
+exports.getTrickOrTreat = (userId) => {
+  return db.get('trickortreats').find({ userId: userId }).value()
+}
+
+// Create a trick or treat
+exports.createTrickOrTreat = (userId) => {
+  db.get('trickortreats').push({ userId: userId, tricks: 0, treats: 0 }).write()
+}
+
+// Trick a user
+exports.trick = (userId) => {
+  const trickOrTreat = db.get('trickortreats').find({ userId: userId })
+
+  trickOrTreat.assign({ tricks: trickOrTreat.value().tricks++, lastPlayed: new Date() }).write()
+
+  return this.geTrickOrTreat(userId)
```

Comment:
A `t` is missing

```js
this.getTrickOrTreat(userId)
```

Reply:
Yes, also the `treat` function was updating `tricks`? 😕